terraform/production/selection_rules.json

```diff
@@ -0,0 +1,53 @@
+{
+    "rules": [
+        {
+            "rule-type": "selection",
+            "rule-id": "1",
+            "rule-name": "1",
+            "object-locator": {
+                "schema-name": "%",
+                "table-name": "member"
+            },
+        
+            "rule-action": "include",
+            "filters": []
+        }, 
+        {
+            "rule-type": "selection",
+            "rule-id": "2",
+            "rule-name": "2",
+            "object-locator": {
+                "schema-name": "%",
+                "table-name": "cccontactlink"
```

Comment:
We would need two separate DMS set ups as the tables reside in two different databases. This will reflect on the selection rules - they should only cover tables from one database

Reply:
I've re worked it to have two separate dms tasks, one for each database. It's quite different so may need a second review if that's ok :)